docs/application-connector/02-03-proxy-service.md

```diff
@@ -3,6 +3,8 @@ title: Application Gateway
 type: Architecture
 ---
 
+> **CAUTION:** This document describes the Application Gateway working in the default legacy [mode](#architecture-application-connector-components-application-operator). To read about the Application Gateway working in the alternative **gatewayOncePerNamespace** mode, see [this](#details-application-gateway) document. 
```

Comment:
CAUTION or NOTE? 🤔 
https://kyma-project.io/community/guidelines/content/#formatting-formatting-panels

Reply:
> * The red CAUTION panel to call attention to something critical that can cause inoperable behavior.

I also thought about it, but I decide to go with **CAUTION** because these two modes are incompatible and it's important not to get them mixed up. If you apply what's written there in the wrong mode, I suppose it CAN cause inoperable behavior. Am I correct? 🤔

Reply:
If that's the case - sure 👍